avm/res/compute/virtual-machine/main.bicep

```
@@ -97,6 +97,9 @@ param bootDiagnosticStorageAccountUri string = '.blob.${environment().suffixes.s
 @description('Optional. Resource ID of a proximity placement group.')
 param proximityPlacementGroupResourceId string = ''
 
+@description('Optional. Resource ID of a virtual machine scale set, where the VM should be added.')
+param virtualMachineScaleSetId string = ''
```

```suggestion
param virtualMachineScaleSetResourceId string = ''
```
Note, needs to be update in references as well and requires a refresh of the generated files

---

@AlexanderSehr done